### PR TITLE
4612: success message updates

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

File: web-client/src/presenter/actions/setSuccessFromDocumentTitleAction.js [hunk 1]
```diff
@@ -7,7 +7,18 @@ import { state } from 'cerebral';
  * @param {object} params.get the cerebral get function
  * @returns {object} the props with the message
```
Comment (matthopson) [inline web-client/src/presenter/actions/setSuccessFromDocumentTitleAction.js:9]: store

File: web-client/src/presenter/actions/setSuccessFromDocumentTitleAction.test.js [hunk 2]
```diff
@@ -43,4 +37,25 @@ describe('setSuccessFromDocumentTitleAction,', () => {
 
     expect(result.output.alertSuccess.message).toEqual('Order updated.');
   });
+
+  it('sets the success message if state.isCreatingOrder is true', async () => {
```
Comment (matthopson) [inline web-client/src/presenter/actions/setSuccessFromDocumentTitleAction.test.js]: maybe "sets the created test message" ?

**Review by matthopson — APPROVED** [review]
Couple nits for now or later.

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/presenter/actions/setPropsForRemoveSignatureAction.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { state } from 'cerebral';
+
+/**
+ * sets the document to be edited from the current caseDetail
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.props the cerebral props object
+ * @param {object} providers.store the cerebral store object
+ * @returns {void} sets the documentToEdit on state
+ */
```
Comment (matthopson) [inline web-client/src/presenter/actions/setPropsForRemoveSignatureAction.js:9]: jsdocs

File: web-client/src/presenter/actions/FileDocument/getFileExternalDocumentAlertSuccessAction.js [hunk 4]
```diff
@@ -7,8 +7,13 @@ import { state } from 'cerebral';
  * @param {object} providers.props the cerebral props object
  * @returns {object} the alertSuccess object with default strings
  */
-export const getFileExternalDocumentAlertSuccessAction = ({ get, props }) => {
+export const getFileExternalDocumentAlertSuccessAction = ({
+  get,
+  props,
+  store,
```
Comment (matthopson) [inline web-client/src/presenter/actions/FileDocument/getFileExternalDocumentAlertSuccessAction.js:15]: missing from jsdocs

File: web-client/src/presenter/actions/skipSigningOrderAction.js [hunk 5]
```diff
@@ -6,9 +6,20 @@ import { state } from 'cerebral';
  * @param {object} providers the providers object
  * @param {object} providers.get the cerebral get function
  *
```
Comment (matthopson) [inline web-client/src/presenter/actions/skipSigningOrderAction.js]: missing `store`

**Review by matthopson — APPROVED** [review]

**Pull request merged**